src/Markdig/MarkdownParserContext.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+
+namespace Markdig
+{
+    /// <summary>
+    /// Provides a context that can be used as part of parsing Markdown documents.
+    /// </summary>
+    public sealed class MarkdownParserContext
+    {
+        /// <summary>
+        /// Gets or sets the context property collection.
+        /// </summary>
+        public IDictionary<object, object> Properties { get; set; }
```

Comment:
You can expose only a `Dictionary<object, object>`, not a `IDictionary` and remove also the `setter`.

Reply:
Nevermind, forgot that I can do that right away from github, so simple 😅